### PR TITLE
Update configuration file generation

### PR DESCRIPTION
**Pull request by jvanulde**
Script now takes a template containing all known layers and removes what doesn't exist on Elasticsearch.

### CODE REVIEW

**Review by anthonyfok — APPROVED** [review]
Looks great!  Definitely more clean with the separated template file.  👍

File: opendrr_config_template.yml [hunk 1]
```diff
@@ -0,0 +1,1443 @@
...
+    #         temporal:
+    #             begin: 2020-08-06
+    #             end: null # or empty (either means open ended)
+    #     providers:
+    #         - type: feature
+    #           name: Elasticsearch
+    #           data: https://api.stage.riskprofiler.ca/opendrr_psra_src_loss
+    #           id_field: Sauid
+    
+    opendrr_psra_hmaps:
```
Comment (drotheram) [inline opendrr_config_template.yml:1316]: hmaps and uhs have been removed

Comment (jvanulde) [reply opendrr_config_template.yml:2119]: They won't show up, they get stripped out of the template by the script.

**Review by drotheram — APPROVED** [review]
Looks good

**Pull request merged**